api/src/pcapi/alembic/versions/20230509T145346_ddd0895ce8c6_create_offer_validation_rule_and_sub_rule_tables.py

```diff
@@ -0,0 +1,100 @@
+"""create_offer_validation_rule_and_sub_rule_tables
```

Comment:
(détail) Un poil plus lisible dans l'historique Alembic : 

```suggestion
"""Create "offer_validation_rule" and "offer_validation_sub_rule" tables
```

;)